tests/property/ic_api/trap/src/index.ts

```diff
@@ -0,0 +1,56 @@
+// TODO to improve these tests we would also test init, postUpgrade, and preUpgrade
```

Comment:
could you briefly explain why you didn't

Reply:
I'll just remove this commented code

Reply:
Removed